pkg/ccl/changefeedccl/enriched_source_provider.go

```diff
@@ -61,13 +62,33 @@ func (p *enrichedSourceProvider) Schema() (*avro.FunctionalRecord, error) {
 
 func (p *enrichedSourceProvider) GetJSON(row cdcevent.Row) (json.JSON, error) {
 	// TODO(various): Add fields here.
-	keys := []string{"job_id"}
+	keys := []string{
```

Comment:
can you update `getAvroFields` / `avroSourceFunction`? ideally you'd have them not able to fall out of sync with each other